src/GrantDatabase.CommandLine/Commands/Settings/AlterRoleSettings.cs

```diff
@@ -10,5 +11,9 @@ public class AlterRoleSettings
 
         [CommandArgument(4, "<Role Name>")]
         public string RoleName { get; set; } = string.Empty;
-    }
+
+		[CommandOption("--create")]
+		[DefaultValue(false)]
+		public bool CreateIfNotExists { get; set; }
```

Comment:
Wasn't sure if you think it might be better to have a 'CreateRoleCommand' instead of this feature.

Reply:
This works great. It mirrors most SQL functionality anyways (in terms of create if not exist)